scripts/move_wcs.py

```diff
@@ -0,0 +1,97 @@
+import glob
+from astropy.io import fits
+from astropy import wcs
+from astropy.wcs import InvalidTransformError
+import six
+
+"""
+More Kw to consider 
+radesys - I think it should be moved - "OTHER" in HST
+velocity aberrration related kw - move them
+  DVA_RA< DVA_DEC, VA_SCALE
+
+TARG_RA, TARG_DEC
+"""
+stsci_wcs_kw = ['PA_V3', 'V2_REF', 'V3_REF', 'PA_APER', 'VPARITY',
```

Comment:
What about WCSAXES and V3I_YANG? Or is WCSAXES handled automatically along with the other FITS standard WCS keywords?